internal/safefile/safeopen.go

```diff
@@ -135,8 +74,8 @@ func openRelativeInternal(path string, root *os.File, accessMask uint32, shareFl
 	oa.Length = unsafe.Sizeof(oa)
 	oa.ObjectName = upathBuffer
 	oa.RootDirectory = uintptr(root.Fd())
-	oa.Attributes = _OBJ_DONT_REPARSE
-	status := ntCreateFile(
+	oa.Attributes = winapi.OBJ_DONT_REPARSE
+	status := winapi.NTCreateFile(
```

Comment:
Given the low-level-bindings intent of `winapi`, I think keeping the Windows naming (`NtCreateFile` vs `NTCreateFile`) will be more consistent.